docs/api/alerting/create.asciidoc

```diff
@@ -0,0 +1,200 @@
+++++
+
+Create {kib} rules.
+
+[[rules-create-api-request]]
+==== Request
+
+`POST <kibana host>:<port>/api/alerting/rule/<id>`
+
+`POST <kibana host>:<port>/s/<space_id>/api/alerting/rule/<id>`
+
+[[rules-create-api-path-params]]
+==== Path parameters
+
+`<id>`::
+  (Optional, string) Specifies a UUID v1 or v4 to use instead of a randomly generated ID.
+
+`space_id`::
+  (Optional, string) An identifier for the space. If `space_id` is not provided in the URL, the default space is used.
+
+[[rules-create-api-request-body]]
+==== Request body
+
+`name`::
+  (Required, string) A name to reference and search.
+
+`tags`::
+  (Optional, string array) A list of keywords to reference and search.
+
+`rule_type_id`::
+  (Required, string) The ID of the rule type that you want to call when the rule is scheduled to run.
+
+`schedule`::
+  (Required, object) The schedule specifying when this rule should be run, using one of the available schedule formats specified under
++
+._Schedule Formats_.
+[%collapsible%open]
+=====
+A schedule is structured such that the key specifies the format you wish to use and its value specifies the schedule.
+
+We currently support the _Interval format_ which specifies the interval in seconds, minutes, hours or days at which the rule should execute.
+Example: `{ interval: "10s" }`, `{ interval: "5m" }`, `{ interval: "1h" }`, `{ interval: "1d" }`.
+
+There are plans to support multiple other schedule formats in the near future.
+=====
+
+`throttle`::
+  (Optional, string) How often this rule should fire the same actions. This will prevent the rule from sending out the same notification over and over. For example, if a rule with a `schedule` of 1 minute stays in a triggered state for 90 minutes, setting a `throttle` of `10m` or `1h` will prevent it from sending 90 notifications during this period.
+
+`notify_when`::
+  (Required, string) The condition for throttling the notification: `onActionGroupChange`, `onActiveAlert`, or `onThrottleInterval`.
+
+`enabled`::
+  (Optional, boolean) Indicates if you want to run the rule on an interval basis after it is created.
+
+`consumer`::
+  (Required, string) The name of the application that owns the rule. This name has to match the Kibana Feature name, as that dictates the required RBAC privileges.
+
+`params`::
+  (Required, object) The parameters to pass to the rule type executor `params` value. This will also validate against the rule type params validator, if defined.
+
+`actions`::
+  (Optional, object array) An array of the following action objects.
++
+.Properties of the action objects:
+[%collapsible%open]
+=====
+  `group`:::
+    (Required, string) Grouping actions is recommended for escalations for different types of alerts. If you don't need this, set this value to `default`.
+
+  `id`:::
+    (Required, string) The ID of the connector saved object to execute.
+
+  `connector_type_id`:::
+    (Required, string) The ID of the <<action-types,connector type>>.
+
```

Comment:
This field is no longer accepted  in the new API.

Reply:
Fixed in https://github.com/elastic/kibana/pull/94745/commits/bad0757c542ca9f6391d3eb6ad01d38e00b5e892.